x-pack/plugins/enterprise_search/server/lib/engines/field_capabilities.ts

```diff
@@ -22,13 +25,134 @@ export const fetchEngineFieldCapabilities = async (
     include_unmapped: true,
     index: getEngineIndexAliasName(name),
   });
+  const fields = parseFieldsCapabilities(fieldCapabilities);
   return {
     created,
     field_capabilities: fieldCapabilities,
+    fields,
     name,
     updated,
   };
 };
 
 // Note: This will likely need to be modified when engines move to es module
 const getEngineIndexAliasName = (engineName: string): string => `search-engine-${engineName}`;
+
+export const parseFieldsCapabilities = (fieldCaps: FieldCapsResponse): SchemaField[] => {
+  const { fields } = fieldCaps;
+  return Object.entries<Record<string, FieldCapsFieldCapability>>(fields)
+    .filter(([fieldName]) => isTopLevelField(fieldName))
+    .map(([fieldName, fieldValue]) => ({
+      ...parseFieldCapability(
+        fieldValue,
+        fieldCaps.indices,
+        getSubFieldCapabilities(fieldName, fields)
+      ),
+      name: fieldName,
+    }));
+};
+
+export const isTopLevelField = (fieldName: string): boolean => {
+  if (fieldName.startsWith('_')) return false;
```

Comment:
Where did we land on metadata fields starting with `_`? I thought we expected some of them to be included in schema display?

Reply:
we're going to include meta fields. i'll remove this filter